src/main/resources/com/adobe/jenkins/github_pr_comment_build/TriggerPRReviewBranchProperty/config.jelly

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    <f:entry title="Allow Untrusted users to trigger the build" field="allowUntrusted">
```

Comment:
One more tiny change: `Untrusted` -> `untrusted` for consistency with casing. May need another screenshot, but I doubt it'll bother anyone if that's different.

Reply:
no biggie

Reply:
and no more screenshots :)